src/main/java/com/test/controller/OrganisationController.java

```diff
@@ -1,69 +0,0 @@
-package com.test.controller;
-
-import java.net.URI;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.DeleteMapping;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.PutMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
-
-import com.test.model.Employee;
-import com.test.model.Organisation;
-import com.test.service.OrganisationService;
-
-@RestController
-@RequestMapping("/organisation")
-public class OrganisationController {
-
-	@Autowired
-	OrganisationService orgService;
```

Comment:
@Rinshakp 

The access modifier is not defined while using the dependency injection.